package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node src/bin/www",
+    "start": "tsc && node dist/bin/www",
```

Comment:
tsc should be part of the build script and not part of the start script.
So:
"build": "tsc",
"start": "node dist/bin/www"

Reply:
could also add a "buildAndStart" script which does as you have the start script here

Reply:
Definitely agree, will update